docs/examples/notebooks/visualization/snippet.py

```diff
@@ -0,0 +1,631 @@
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+
+import plotly.graph_objects as go
+import itertools
+import numpy as np
+from pathlib import Path
+from typing import Optional
+
+import itertools
+import warnings
+from abc import ABC, abstractmethod
+from enum import Enum
+from functools import cached_property
+from pathlib import Path
+from typing import Optional, Set, Tuple, Type
+
+import matplotlib.pyplot as plt
+import numpy as np
+from deltakit_circuit import Qubit
+from deltakit_circuit.basic_types import Coord2D, Coord2DDelta
+from deltakit_circuit.qubit_identifiers import PauliGate, PauliX, PauliZ
+from deltakit_explorer.codes.css._css_code import CSSCode
+from deltakit_explorer.codes.stabiliser import Stabiliser
+from deltakit_explorer.enums.basic_enums import DrawingColours
```

Comment:
I needed these to make this run with `main`. 
```suggestion
from deltakit_circuit._basic_types import Coord2D, Coord2DDelta
from deltakit_circuit._qubit_identifiers import PauliGate, PauliX, PauliZ
from deltakit_explorer.codes._css._css_code import CSSCode
from deltakit_explorer.codes._stabiliser import Stabiliser
from deltakit_explorer.enums._basic_enums import DrawingColours
```

Also, my IDE says there a lot of unused imports (gray).
<img width="597" height="563" alt="image" src="https://github.com/user-attachments/assets/df40c8b0-a5b7-4438-b029-00049ac65114" />

Can these be removed?

Reply:
That looks like a lot of private imports, would it make sense to use the public-facing imports instead?

Reply:
Yeah, good idea. I was going to mention it, too, but `DrawingColors` isn't public and a almost all thre rest can be removed. But after that, it would still be good to import `PauliX` from `deltakit.circuit` directly.